test/udp_backend_test.rb

```diff
@@ -132,4 +132,16 @@ def test_io_error_should_not_raise_but_log
     @logger.expects(:error)
     StatsD.increment('fail')
   end
+
+  def test_thread_error_does_not_raise
```

Comment:
not sure I understand this test... shouldn't there be some kind of test assertion? or is this basically just testing that nothing raises? what did it raise before?